### ISSUE TITLE
Verify lzop is installed when doing compressed transfers

### ISSUE THREAD

**Issue by tytgatlieven** [masc3d/btrfs-sxbackup#48]
Dear

First of all, thanks a lot for the great software! It's amazing.

I had to reinstall my server and forgot to install lzop, which lead to some figuring out why the snapshot using compression didn't work.

Not a huge problem, just something I noticed

Kind regards, and keep up the good work!

Lieven

**Comment by masc3d** [comment]
yes, there could be some more explicit checks and feedback in quite a few places.

I avoided this initially as every single command executed remotely initiates a new ssh connection, which is expensive so I'm keeping roundtrips to a minimum.
Explicitly checking for `lzop` eg. would be a single roundtrip just or that.

However there's a possibility using ssh's `ControlMaster` to multiplex, so once there's support for that I do agree this should be implemented.
